client/js/components/StatsLeaderboard.vue

```diff
@@ -0,0 +1,359 @@
+<template>
+  <div>
+    <div id="stats-table-upper-surface" class="d-flex align-end">
+      <v-select
+        v-model="selectedMetric"
+        :items="metricChoices"
+        label="Select Metric"
+        class="filter-select mr-4 fit"
+        data-cy="metric-select"
+      />
+      <v-select
+        v-model="selectedWeeks"
+        :items="weeks"
+        label="Select Weeks"
+        data-cy="week-select"
+        class="flex-grow-1"
+        menu-props="data-week-select-menu"
+        multiple
+        chips
+        deletable-chips
+      />
+    </div>
+    <v-data-table
+      :items="tableRows"
+      :headers="tableColumns"
```

Comment:
See [vuetify docs for data tables](https://vuetifyjs.com/en/api/v-data-table/). 

1. Expects `headers` to be array of objects with the `text` to display and the `value` to compare to
2. Expects `items` to be an array of objects with keys that match the header `values` and values for what to display in the table

The `tableRows` computed prop adds additional key/value pairs to each object which are not displayed in order for the templates inside the cells to refer to information about both the points and the wins for the given week.

Reply:
This is all great information, part of me feels like it would be really nice to have as a comment in the template but since this is a vuetify thing it's probably part of their documentation already.

Reply:
Yeah it's pretty well documented in vuetify

Reply:
Perhaps a docs link in comments though